src/Zap.sol

```diff
@@ -361,6 +361,7 @@ contract Zap is Reentrancy, Errors {
         uint128 _collateralId,
         uint256 _collateralAmount,
         address _collateral,
+        bytes memory _path,
```

Comment:
please update natspec with param too 🙏

Reply:
resolved in [bd7f513](https://github.com/JaredBorders/zap/pull/50/commits/bd7f513596aa172a4894e233e83978690e373e55)